src/main/java/org/folio/bulkops/service/LogFilesService.java

```diff
@@ -92,4 +70,34 @@ public void deleteFileByOperationIdAndName(UUID operationId, String fileName) {
     remoteFileSystemClient.remove(csvPath, jsonPath);
     log.info("Deleted: {}, {}", csvPath, jsonPath);
   }
-}
+
+  public void removeTriggeringAndMatchedRecordsFiles(BulkOperation bulkOperation) {
+    if (isNotEmpty(bulkOperation.getLinkToTriggeringCsvFile())) {
```

Comment:
Is it possible to face with case when file is presented but link is null? If yes it makes sense to remove isNotEmpty verification.

Reply:
Additionally, it needs to verify behavior remoteFileSystemClient.remove(null) if validation isNotEmpty will be removed.